mkosi/__init__.py

```diff
@@ -2678,8 +2678,12 @@ def run_pacman(packages: Set[str]) -> None:
     with mount_api_vfs(args, root):
         run_pacman(packages)
 
-    with open(os.path.join(root, "etc/locale.gen"), "w") as f:
-        f.write("en_US.UTF-8 UTF-8\n")
+    def _enable_locale(line: str) -> str:
+        if line.startswith("#en_US.UTF-8"):
+            return line.replace("#", "")
+        return line
+
```

Comment:
This assumes that /etc/locale.gen exists and that the locale will always be commented in /etc/locale.gen. While I don't think that's going to change anytime soon, let's make this a bit more future proof by only patching /etc/locale.gen if it and appending the locale to the file if it doesn't exist as a commented locale.

Reply:
@DaanDeMeyer Thanks for the suggestion !
According to your proposition, I’ve just updated the PR to make `mkosi` uncomment the locale in `/etc/locale.gen` if the file exists and to create this file and write the locale in it if it doesn’t exist.